spring-context/src/main/java/org/springframework/cache/annotation/Cacheable.java

```diff
@@ -137,6 +137,7 @@
 	 * can be accessed via {@code #root.args[1]}, {@code #p1} or {@code #a1}. Arguments
 	 * can also be accessed by name if that information is available.</li>
 	 * </ul>
+	 * If condition is evaluated to {@code true}, result is cached.
```

Comment:
I think this would be better as an opening statements, rather than at the end. We should also update the paragraph about the default to mention that if no expression is set, the method result is always cached.

Reply:
Nice words for me, thanks. I am on it.

Reply:
Is it better now?
Should I squash / rebase?